adafruit_platformdetect/board.py

```diff
@@ -46,6 +46,7 @@
 ODROID_C2                   = "ODROID_C2"
 
 FTDI_FT232H                 = "FT232H"
+LINARO_96BOARDS             = "LINARO_96BOARDS"
```

Comment:
I opted to use `LINARO_96BOARDS` here instead of _96BOARDS - leading underscores are generally a convention for indicating non-public methods/attributes.  Please let me know if "LINARO" isn't really appropriate.

It's possible we should add a prefix to the board and chip constants - like `BOARD_FOO` / `CHIP_FOO` or somesuch - to deal with the general problem that some things will have a leading digit and Python doesn't like leading digits.

Reply:
Just `96BOARDS` should be fine.
As for prefix, I'm pretty neutral on that. If you do want to add, it's better to add the Chip vendor name like `QCOM_<CHIP_ID>` or `HISI_<CHIP_ID>`

Reply:
Unfortunately we can't use a bare `96BOARDS` because a leading digit on a variable is a syntax error.  I'm inclined to think some generic prefix is probably a good idea, though it will make things more verbose.

Reply:
Oh okay... I thought you were being specific about chip_id.
Well in that case, you could go for `LINARO_96BOARDS` that's fine.